moped-editor/public/index.html

```diff
@@ -36,6 +36,10 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Symbols+Outlined"
+    />
```

Comment:
in order to get the material symbols https://fonts.google.com/icons?selected=Material+Symbols+Outlined:menu_book:FILL@0;wght@400;GRAD@0;opsz@48

Reply:
thank you for figuring this out! i didn't realize this required adding another stylesheet. would you mind adding that comment ☝️ into the code? tbh i have mixed feelings about adding in another icon source outside of the standard MUI imports. fwiw MUI v5 has twice as many icons as v4, so i'd like to revisit this if/when we upgrade.